controllers/resources.go

```diff
@@ -224,6 +224,10 @@ func ProduceKubernetesApplication(scheme *runtime.Scheme, wp v1alpha1.WordpressI
 			Template: serviceInRemote,
 		},
 	}
+	targetSelector := &metav1.LabelSelector{}
+	if wp.Spec.ProvisionPolicy == nil || *wp.Spec.ProvisionPolicy == v1alpha1.ProvisionNewCluster {
+		targetSelector.MatchLabels = GetLocalResourceSelector(wp)
```

Comment:
So the target selector is only set if the provision policy is to provision a new cluster . . . is that right? It seems like a target selector would be useful if we're using an existing cluster. I am assuming I am only confused because I don't know enough about this feature yet, and that this is correct for the behavior we want : ).

I reasoned through it a bit more and checked [the docs](https://crossplane.io/docs/master/api/crossplaneio/crossplane/workload-crossplane-io-v1alpha1.html#kubernetesapplicationspec), and for posterity, this does seem correct. An empty target selector will match anything, whereas the local resource selector will match the cluster we're creating.

Reply:
This is spot-on. We made a decision to go this direction in #26 for short-term simplicity. If we want to modify in the future I am very open to that :)